pkg/kev/sources.go

```diff
@@ -86,6 +91,8 @@ func withEnvVars(s *Sources, origin *ComposeProject) error {
 			Name:        svc.Name,
 			Labels:      svc.Labels,
 			Environment: originSvc.Environment,
+			Extensions:  svc.Extensions,
+			K8SConfig:   svc.K8SConfig,
```

Comment:
Do we need K8sConfig here given we already track Extensions?

Reply:
will remove it now, I don't think we want to cache anything